canonical/lxd-ui
# chore: add oidc login tests [WD-9461]

## Done

- Added OIDC secrets to the `ci-end-to-end` github environment to be access in the PR checks action.
- Added ui login test for OIDC identities. This test is run before other e2e tests since we need to remove tls certificates as part of the process.
- update CONTRIBUTING.md with relevant info for e2e tests

## NB!
Once this PR gets merged, we will need to review approve workflow runs as the browser e2e job is using the `ci-end-to-end` environment. 

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Ensure CI passes.
